src/IECoreScene/TriangulateOp.cpp

```diff
@@ -47,48 +47,12 @@ IE_CORE_DEFINERUNTIMETYPED( TriangulateOp );
 
 TriangulateOp::TriangulateOp() : MeshPrimitiveOp( "A MeshPrimitiveOp to triangulate a mesh" )
 {
-	m_toleranceParameter = new FloatParameter(
-		"tolerance",
-		"The floating point tolerance to use for various operations, such as determining planarity of faces",
-		1.e-6f,
-		0.0f
-	);
-
-	m_throwExceptionsParameter = new BoolParameter(
-		"throwExceptions",
-		"When enabled, exceptions are thrown when invalid geometry is encountered (e.g. non-planar or concave faces).",
-		true
-	);
-
-
-	parameters()->addParameter( m_toleranceParameter );
-	parameters()->addParameter( m_throwExceptionsParameter );
 }
 
 TriangulateOp::~TriangulateOp()
 {
 }
 
-FloatParameter * TriangulateOp::toleranceParameter()
-{
-	return m_toleranceParameter.get();
-}
-
-const FloatParameter * TriangulateOp::toleranceParameter() const
-{
-	return m_toleranceParameter.get();
-}
-
-BoolParameter * TriangulateOp::throwExceptionsParameter()
-{
-	return m_throwExceptionsParameter.get();
-}
-
-const BoolParameter * TriangulateOp::throwExceptionsParameter() const
-{
-	return m_throwExceptionsParameter.get();
-}
```

Comment:
I didn't catch this before merging, but these removals should have been accompanied by (a) a Breaking Change callout in the commit log and (b) removing the methods and members from `TriangulateOp.h` as well.

Reply:
Same goes for `MeshPrimitiveShrinkWrapOp` I presume

Reply:
FWIW it may have been better to simply remove TriangulateOp. It’s been deprecated a while now, and then I’d have caught the problem in my client code at compile time rather than runtime.